[2024-09-09T20:35:07.973-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-09T20:35:07.986-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T23:14:21.140101+00:00 [queued]>
[2024-09-09T20:35:07.993-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T23:14:21.140101+00:00 [queued]>
[2024-09-09T20:35:07.994-0300] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-09-09T20:35:08.005-0300] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): dag_dbt_raw_model> on 2024-09-09 23:14:21.140101+00:00
[2024-09-09T20:35:08.008-0300] {standard_task_runner.py:72} INFO - Started process 235819 to run task
[2024-09-09T20:35:08.010-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'simple_dbt_dag', 'dag_dbt_raw_model', 'manual__2024-09-09T23:14:21.140101+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt_raw_model.py', '--cfg-path', '/tmp/tmp4rl4do7l']
[2024-09-09T20:35:08.011-0300] {standard_task_runner.py:105} INFO - Job 5: Subtask dag_dbt_raw_model
[2024-09-09T20:35:08.056-0300] {task_command.py:467} INFO - Running <TaskInstance: simple_dbt_dag.dag_dbt_raw_model manual__2024-09-09T23:14:21.140101+00:00 [running]> on host DESKTOP-0CT35O1.
[2024-09-09T20:35:08.116-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='simple_dbt_dag' AIRFLOW_CTX_TASK_ID='dag_dbt_raw_model' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T23:14:21.140101+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-09T23:14:21.140101+00:00'
[2024-09-09T20:35:08.118-0300] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-09-09T20:35:08.119-0300] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-09-09T20:35:08.120-0300] {logging_mixin.py:190} INFO - Current task name:dag_dbt_raw_model state:running start_date:2024-09-09 23:35:07.988174+00:00
[2024-09-09T20:35:08.122-0300] {logging_mixin.py:190} INFO - Dag name:simple_dbt_dag and current dag run status:running
[2024-09-09T20:35:08.123-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-09T20:35:08.125-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T20:35:08.126-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "cd '/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/dbt_project/bancos_airflow' && pyenv activate dbt_env && dbt run --models bancos_raw"]
[2024-09-09T20:35:08.132-0300] {subprocess.py:86} INFO - Output:
[2024-09-09T20:35:08.164-0300] {subprocess.py:93} INFO - /usr/bin/bash: line 1: pyenv: command not found
[2024-09-09T20:35:08.165-0300] {subprocess.py:97} INFO - Command exited with return code 127
[2024-09-09T20:35:08.176-0300] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T20:35:08.190-0300] {logging_mixin.py:190} INFO - Task instance in failure state
[2024-09-09T20:35:08.192-0300] {logging_mixin.py:190} INFO - Task start:2024-09-09 23:35:07.988174+00:00 end:2024-09-09 23:35:08.190496+00:00 duration:0.202322
[2024-09-09T20:35:08.193-0300] {logging_mixin.py:190} INFO - Task:<Task(BashOperator): dag_dbt_raw_model> dag:<DAG: simple_dbt_dag> dagrun:<DagRun simple_dbt_dag @ 2024-09-09 23:14:21.140101+00:00: manual__2024-09-09T23:14:21.140101+00:00, state:running, queued_at: 2024-09-09 23:14:21.153822+00:00. externally triggered: True>
[2024-09-09T20:35:08.194-0300] {logging_mixin.py:190} INFO - Failure caused by Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T20:35:08.195-0300] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=simple_dbt_dag, task_id=dag_dbt_raw_model, run_id=manual__2024-09-09T23:14:21.140101+00:00, execution_date=20240909T231421, start_date=20240909T233507, end_date=20240909T233508
[2024-09-09T20:35:08.207-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-09T20:35:08.208-0300] {standard_task_runner.py:124} ERROR - Failed to execute job 5 for task dag_dbt_raw_model (Bash command failed. The command returned a non-zero exit code 127.; 235819)
Traceback (most recent call last):
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
  File "/mnt/c/Users/teu20/Documents/Poli/Repo/eEDB011-20243/atividade 6 - WSL/airflow/airflow_env/lib/python3.10/site-packages/airflow/operators/bash.py", line 249, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2024-09-09T20:35:08.264-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-09-09T20:35:08.278-0300] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-09T20:35:08.280-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
